scripts/labs/argocd/kind-argo.sh

```diff
@@ -5,7 +5,6 @@ set -e
 readonly SRC=$(dirname "$0")
 
 $SRC/common-steps.sh
-$SRC/6-metrics-server.sh
```

Comment:
why did you change that ?

Reply:
Well is seemed to be part of the "common" stuff?

Reply:
LGTM? what does that mean?

Reply:
LGTM = looks good to me ;)

Reply:
What ever you decide is common.

Reply:
I don’t remember the exact reason but I would prefer we don’t change it (at least not in this PR)